src/party.cpp

```diff
@@ -432,8 +434,9 @@ bool Party::canUseSharedExperience(const Player* player) const
 		}
 	}
 
-	uint32_t minLevel = static_cast<int32_t>(std::ceil((static_cast<float>(highestLevel) * 2) / 3));
```

Comment:
Can anyone explain me why we are casting this expression here to ``int32_t`` and then assigning to a ``uint32_t``? @kornholi @ranisalt @djarek

Reply:
No, the way you did is ok. The static cast is implicit, though, but keep it that way :)